pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -103,7 +103,7 @@ function LoadPackageAndExposeGlobals(package::String, mod::String; all_globals::
     Base.invokelatest(LoadPackageAndExposeGlobals, package, mod_mod; all_globals = all_globals)
 end
 
-function LoadPackageAndExposeGlobals(package::String, mod::Module; all_globals::Bool = false)
+function LoadPackageAndExposeGlobals(package::String, mod::Module; all_globals::Bool = false, overwrite::Bool = false)
```

Comment:
Can we please have documentation for this function? It's getting more and more complicated.

Also, I think it should be moved to a separate file, afterwards I think we could rename `gap.jl` to `adapter.jl` (to match `adapter.gi`). But of course that can happen after this PR resp. after PR #249, I just wanted to mention it.

Reply:
The documentation is above. I personally think the function should be used at all, but it is handy to have it for using GAP from Julia, if you only want to use GAP.